src/components/modules/CodeRecovery/modules/RecoveryPhone/modules/WithPhone/specs/WithPhone.view.spec.js

```diff
@@ -75,7 +75,7 @@ describe('WithPhone', () => {
 
         expect(
           wrapper.find('[data-test=code-input]').attributes().error,
-        ).toBeUndefined();
+        ).toBeFalsy();
```

Comment:
Why?

Reply:
because error prop in ui-input default prop is not null now, but empty string `''`